main/utils/autorun/schedule.js

```diff
@@ -0,0 +1,57 @@
+const { join } = require('path')
+const { initConfigFile } = require('../config')
+const { getAutorunHomeDir } = require('../paths')
+
+// Initialize OONI_HOME for autorun by generating a fresh config file
+// in the OONI_HOME for autorun
+// Note: initConfigFile in turn initializes Sentry (again) with no extra effect
+initConfigFile({ configFilePath: join(getAutorunHomeDir(), 'config.json') })
```

Comment:
How is the config file for the autorun kept in sync with the config file for the normal OONI Probe?

Reply:
I lost sight of all the usecases for which we wanted to `sync` and not `copy`. I have to do this now.